packages/edit-site/src/components/sidebar-navigation-screen-page/index.js

```diff
@@ -18,6 +33,78 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import { unlock } from '../../private-apis';
 import { store as editSiteStore } from '../../store';
 import SidebarButton from '../sidebar-button';
+import SidebarNavigationSubtitle from '../sidebar-navigation-subtitle';
+import SidebarDetails from '../sidebar-navigation-data-list';
+import DataListItem from '../sidebar-navigation-data-list/data-list-item';
+import StatusLabel from './status-label';
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+function getPageDetails( page ) {
+	if ( ! page ) {
+		return [];
+	}
+	const details = [];
+
+	if ( page?.status ) {
+		details.push( {
+			label: __( 'Status' ),
+			value: <StatusLabel status={ page.status } date={ page?.date } />,
+		} );
+	}
+
+	if ( page?.slug ) {
+		details.push( {
+			label: __( 'Slug' ),
+			value: <Truncate numberOfLines={ 1 }>{ page.slug }</Truncate>,
+		} );
+	}
+
+	if ( page?.templateTitle ) {
+		details.push( {
+			label: __( 'Template' ),
+			value: page.templateTitle,
+		} );
+	}
+
+	details.push( {
+		label: __( 'Parent' ),
+		value: page?.parentTitle,
+	} );
+
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = page?.content?.raw
```

Comment:
I think we should use the `rendered` content and not the `raw` one(shortcodes etc..).